### ISSUE TITLE
interface UpdateManager 方法返回值未定义

### ISSUE THREAD

**Issue by shenopkss** [Emeryao/typed-we-app#22]
报错信息
```
node_modules/@types/we-app/index.d.ts(1062,9): error TS7010: 'onCheckForUpdate', which lacks return-type annotation, implicitly has an 'any' return type.
node_modules/@types/we-app/index.d.ts(1063,9): error TS7010: 'onUpdateFailed', which lacks return-type annotation, implicitly has an 'any' return type.
node_modules/@types/we-app/index.d.ts(1064,9): error TS7010: 'onUpdateReady', which lacks return-type annotation, implicitly has an 'any' return type.
```
修改代码后通过
```
interface UpdateManager {
        applyUpdate: Function,
        onCheckForUpdate(callback: Function):any,
        onUpdateFailed(callback: Function):any,
        onUpdateReady(callback: Function):any
    }
```

**Comment by emeryao** [comment]
Thanks for your issue :+1:
Could you open a PR to fix it?

**Comment by emeryao** [comment]
published to npm with version `1.7.0-update.9`
